### ISSUE TITLE
change picture dimension

### ISSUE THREAD

**Issue by mhamri** [greenshot/greenshot#127]
time to time I need to attach two different screenshots side by side, atm the Greenshot keep the size of the screen exactly to the size of the picture and can't add another picture next to it. 

or sometimes I need to write some text around the screenshot and not on it. in this way i need to capture the screenshot and copy it into the Ms Paint and then write next to it. but if i could change the size of the screen in the app or add extra whitespace around it it would be perfect

**Comment by jinek** [comment]
May be it's better to just add a button "Insert from clipboard at right" ?

**Comment by FF-Brown** [comment]
Working on this feature. This is something I've wanted for awhile as well.

**Comment by mischlrebl** [comment]
it is already possible to add extra whitespace around:

In order to enlarge the screenshot, hit **Shift** + **+**, which will add 25 pixels to all four sides

source: https://getgreenshot.org/help/

**Comment by FF-Brown** [comment]
True, there is that work-around. It works, but it's not super efficient, since in most cases you'll have to add space several times, move the object, and then crop the extra space from the other three sides. 
I'm gonna move forward with a context-menu option to shift an object off to one side of the main screenshot.

**Comment by jvl1v5** [comment]
Looking forward to this feature too. 

In some cases a button for shifting the object to one side or the other could be some of a limitation if you have to arrange multiple screenshots.

Another way to work with this could be to have a button to open a "blank page" with a defined background color. 
Currently I work around this with a fullscreen screenshot, draw a colored rectangle and than paste the screenshots I need to arrange.

Perhaps you could build a "new page" menu where you can define the sizes of the new page/picture and a background color.

I would imagine that is a more simple solution to implement?

**Comment by FF-Brown** [comment]
There's a PR open for these changes! A couple years later hahaha. This was on my backburner for a long time, and after the changes were implemented I got busy again and didn't put a PR up until recently.

Feel free to experiment with the code, and leave comments/thumbs up to show support so the owners know this is something people are excited about!

**Comment by hennenzac** [comment]
Hi @FF-Brown , do you know what PR that is?  I can't seem to find it and I'd really like to try this feature.  Thank you.

**Comment by FF-Brown** [comment]
@hennenzac Yeah, I guess I could've linked it in my comment. I didn't expect the review to still be open after so long, though. 
This is the PR: https://github.com/greenshot/greenshot/pull/473

**Comment by hennenzac** [comment]
@FF-Brown, wow that was a fast response!  Thank you.  So i'm no expert in all this, but I'm assuming there's no simple installer for this until it's officially released, correct?  I'd have to load your code and run it somehow.  I know some coding but nothing related to an application like this.

**Comment by FF-Brown** [comment]
@hennenzac Yeah, you'd have to open it in visual studio and build it to test it out. Not too difficult. And to use it long-term you'd have to build an installer, bit more annoying. If you're not in a rush I can just send you an installer when I get home tho, I've got one I made awhile ago.

**Comment by FF-Brown** [comment]
@hennenzac Here's the installer. Looks like it contains changes from a couple other PRs as well, but that won't interfere with you trying out this feature.
Promise it's not malware, lol.
https://drive.google.com/drive/folders/18AyHNp5zniwKhC9VlMcsaR2xVWq7ziUo?usp=sharing

**Comment by hennenzac** [comment]
@FF-Brown  Wow!  Thank you very much!  It's working great so far, and I get to play with the other features like the OCR which is really cool.  I wish this stuff was in the official release.  Thanks again.
